<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AVAJ" failures="1" tests="4" name="com.wiproJuly.LabSession18thAug.LabAssignment1" time="33.452" errors="0" timestamp="2025-08-19T16:13:41 GMT+05:30" skipped="0">
  <testcase classname="com.wiproJuly.LabSession18thAug.LabAssignment1" name="testcase1" time="0.002"/>
  <testcase classname="com.wiproJuly.LabSession18thAug.LabAssignment1" name="testcase2" time="0.003"/>
  <testcase classname="com.wiproJuly.LabSession18thAug.LabAssignment1" name="lab1" time="33.444">
    <failure message="expected [Techlistic - Atom ] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Techlistic - Atom ] but found []
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.wiproJuly.LabSession18thAug.LabAssignment1.lab1(LabAssignment1.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- lab1 -->
  <testcase classname="com.wiproJuly.LabSession18thAug.LabAssignment1" name="testcase3" time="0.003"/>
</testsuite> <!-- com.wiproJuly.LabSession18thAug.LabAssignment1 -->
